docs/pyserini.md

```diff
@@ -1,19 +1,46 @@
 # Pyserini: Anserini Integration with Python
 
-Anserini was designed with Python integration in mind, for connecting with popular deep learning toolkits such as PyTorch. This is accomplished via [pyjnius](https://github.com/kivy/pyjnius). The `SimpleSearcher` class provides a simple Python/Java bridge, shown below:
+Anserini was designed with Python integration in mind, for connecting with popular 
+deep learning toolkits such as PyTorch. 
+[Pyserini](https://github.com/castorini/anserini/src/main/python/pyserini) 
+provides a Python interface via [pyjnius](https://github.com/kivy/pyjnius) 
+for accessing various classes within Anserini.
+
+This is an ongoing effort, and contributions for extending the interface are welcome!
+You can also interact with Anserini's Java classes directly 
+using `pyjnius`, as described 
+[here](#Direct-interaction-via-Pyjnius).
```

Comment:
"Capitalize Initial Word of Every Heading" - and below also, e.g., "Setup for using Pyserini" -> "Setup for Using Pyserini"